dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/event/EventSearchParams.java

```diff
@@ -467,58 +467,58 @@ public EventSearchParams setEventStatus( EventStatus eventStatus )
         return this;
     }
 
-    public Date getLastUpdatedStartDate()
+    public Date getUpdatedAtStartDate()
     {
-        return lastUpdatedStartDate;
+        return updatedAtStartDate;
     }
 
-    public EventSearchParams setLastUpdatedStartDate( Date lastUpdatedStartDate )
+    public EventSearchParams setUpdatedAtStartDate( Date updatedAtStartDate )
     {
-        this.lastUpdatedStartDate = lastUpdatedStartDate;
+        this.updatedAtStartDate = updatedAtStartDate;
         return this;
     }
 
-    public Date getLastUpdatedEndDate()
+    public Date getUpdatedAtEndDate()
     {
-        return lastUpdatedEndDate;
+        return updatedAtEndDate;
     }
 
-    public EventSearchParams setLastUpdatedEndDate( Date lastUpdatedEndDate )
+    public EventSearchParams setUpdatedAtEndDate( Date updatedAtEndDate )
     {
-        this.lastUpdatedEndDate = lastUpdatedEndDate;
+        this.updatedAtEndDate = updatedAtEndDate;
         return this;
     }
 
-    public String getLastUpdatedDuration()
+    public String getUpdatedAtDuration()
     {
-        return lastUpdatedDuration;
+        return updatedAtDuration;
     }
 
-    public EventSearchParams setLastUpdatedDuration( String lastUpdatedDuration )
+    public EventSearchParams setUpdatedAtDuration( String updatedAtDuration )
     {
-        this.lastUpdatedDuration = lastUpdatedDuration;
+        this.updatedAtDuration = updatedAtDuration;
         return this;
     }
 
-    public Date getDueDateStart()
+    public Date getScheduleAtStart()
     {
-        return dueDateStart;
+        return scheduleAtStart;
```

Comment:
Should we make this consistent with the `updatedAt` ones which are all `updatedAtStartDate/EndDate`?

So either `scheduleAtStartDate` or remove the `Date` suffix from the other fields/setters/getters